tool/release/createnotes/Commit.kt

```diff
@@ -25,34 +25,36 @@ import com.eclipsesource.json.Json
 import com.vaticle.dependencies.tool.release.createnotes.Constant.github
 import java.nio.file.Path
 
-fun getCommits(org: String, repo: String, current: Version, to: String, baseDir: Path, githubToken: String): List<String> {
-    val preceding = getPrecedingVersion(org, repo, current, githubToken)
+fun collectCommits(org: String, repo: String, commit: String, version: Version, baseDir: Path, githubToken: String): List<String> {
+    println("Determining the commits to be collected...")
+    val preceding = getPrecedingVersion(org, repo, version, githubToken)
     if (preceding != null) {
-        val response = httpGet("$github/repos/$org/$repo/compare/$preceding...$to", githubToken)
+        println("The script will collect commits down to the preceding version '$preceding'.")
+        val response = httpGet("$github/repos/$org/$repo/compare/$preceding...$commit", githubToken)
         val body = Json.parse(String(response.content.readBytes()))
```

Comment:
@lolski 

```suggestion
        val body = Json.parse(response.parseAsString())
```

Reply:
Fixed in 8936646e3586dcfab42e22832c73f7db8e756a65.